src/status_im/ui/screens/routing/core.cljs

```diff
@@ -135,6 +135,27 @@
                                        (bottom-bar/minimize-bar route-name)))))}
            (prepare-config config)))))
 
+(defn twopane-navigator [routes config]
+  (navigation/twopane-navigator
+   routes
+   (cond->
```

Comment:
not sure I get this, mabye some leftover? You are not threading the value, so `cond->` is unnecessary looks like

Reply:
I created `twopane-navigator` by example of `stack-navigator` from @rasom. 
@rasom, could you please take a look, do we still need `cond->` here? Thanks.